server/lib/nl/fulfillment/base.py

```diff
@@ -317,10 +321,12 @@ def perform_existence_check(self):
     else:
       logging.info('Existence check failed for %s - %s', ', '.join(self.places),
                    ', '.join(self.all_svs))
-      self.state.uttr.counters.err('failed_existence_check', {
-          'places': self.places,
-          'svs': list(self.all_svs),
-      })
+      self.state.uttr.counters.err(
+          'failed_existence_check', {
+              'places': self.places[:3],
```

Comment:
i don't understand the slicing here and below. use a constant (or lower the limit of child places used for the existence check?)